lib/ts/recipe/emailverification/recipe.tsx

```diff
@@ -91,16 +84,42 @@ export default class EmailVerification extends RecipeModule<
 
     static init(
         config: UserInput
-    ): CreateRecipeFunction<GetRedirectionURLContext, PreAndPostAPIHookAction, OnHandleEventContext, NormalisedConfig> {
-        return (
-            appInfo: NormalisedAppInfo
-        ): RecipeModule<GetRedirectionURLContext, PreAndPostAPIHookAction, OnHandleEventContext, NormalisedConfig> => {
-            EmailVerification.instance = new EmailVerification({
-                ...config,
-                appInfo,
-                recipeId: EmailVerification.RECIPE_ID,
-            });
-            return EmailVerification.instance;
+    ): RecipeInitResult<
+        GetRedirectionURLContext,
+        PreAndPostAPIHookAction,
+        OnHandleEventContext,
+        NormalisedConfig,
+        PreAndPostAPIHookActionWebJS
+    > {
+        const normalisedConfig = normaliseEmailVerificationFeature(config);
+
+        return {
+            authReact: (
+                appInfo: NormalisedAppInfo
+            ): RecipeModule<
+                GetRedirectionURLContext,
+                PreAndPostAPIHookAction,
+                OnHandleEventContext,
+                NormalisedConfig
+            > => {
+                EmailVerification.instance = new EmailVerification({
+                    ...normalisedConfig,
+                    appInfo,
+                    recipeId: EmailVerification.RECIPE_ID,
+                });
+                return EmailVerification.instance;
+            },
+            webJS: EmailVerificationWebJS.init({
+                ...normalisedConfig,
+                override: {
+                    functions: (originalImpl, builder) => {
+                        const functions = getFunctionOverrides(normalisedConfig.onHandleEvent);
+                        builder.override(functions as any);
```

Comment:
Why do we need this any here?

Reply:
there was an issue i probably just didnt notice i did not remove it after solving the issue